src/hoc/selectable-enhance.js

```diff
@@ -76,12 +76,32 @@ export const SelectableContainerEnhance = (Component) => {
           key: this._keyIndex,
           style: mergedChildrenStyles,
           nestedItems: child.props.nestedItems.map((child) => this._extendChild(child, styles, selectedItemStyle)),
+          initiallyOpen: this._isInitiallyOpen(child),
         });
       } else {
         return child;
       }
     },
 
+    _isInitiallyOpen(child) {
```

Comment:
Honestly, if they're considered private I think we should just take them out of the class and put them at the top of the file.